docker/package/packages.py

```diff
@@ -89,6 +110,28 @@
     node_units.append(SystemdUnit(suffix=network,
                                   service_file=service_file,
                                   startup_script="tezos-node-start"))
+
+# TODO: refactor once #108 is merged
+node_units.append(SystemdUnit(
+    suffix="edo2net",
+    service_file=ServiceFile(Unit(after=["network.target"], requires=[],
+                                  description=f"Tezos node {network}"),
```

Comment:
where is this `network` coming from? :thinking:

Reply:
I assume the last value from the `for` loop above is used here :see_no_evil: